src/pykeen/training/training_loop.py

```diff
@@ -556,7 +556,7 @@ def _train(  # noqa: C901
                     batch_size_sufficient = True
                     logger.info(
                         "Currently automatic memory optimization only supports GPUs, but you're using a CPU. "
-                        "Therefore, the batch_size will be set to the default value '{batch_size}'",
+                        f"Therefore, the batch_size will be set to the default value '{batch_size}'",
```

Comment:
Nice catch, although unrelated 😄 If you want we can also include that into the misc MR here: https://github.com/pykeen/pykeen/pull/1504